recipes/uvw/all/conanfile.py

```diff
@@ -41,12 +40,11 @@ def _supported_compiler(self):
         return False
 
     def requirements(self):
-        if tools.Version(self.version) >= "2.6" and tools.Version(self.version) < "2.7":
-            self.requires("libuv/1.38.1")
+        self.requires("libuv/1.40.0")
```

Comment:
uvw 2.6 is not compatible with libuv 1.40. Each minor version of uvw must be linked to the specific minor version of libuv.

Reply:
Thanks for reviewing.
Can you check my change? I now explicitly check in `package_info`.
Before, it was possible to override the version in a provile or a consumer of uvw.